Extension/src/LanguageServer/configurations.ts

```diff
@@ -808,6 +808,9 @@ export class CppProperties {
                 const matches: string[] = fastGlob.isDynamicPattern(normalized) ?
                     fastGlob.sync(normalized, { onlyDirectories: true, cwd, suppressErrors: true, deep: 15 }) : [res];
                 resolvedGlob.push(...matches.map(s => s + suffix));
+                if (resolvedGlob.length === 0) {
```

Comment:
What is this for? How do I get resolvedGlob to be empty?

Reply:
If the glob pattern doesn't resolve, it pushes an empty string into resolve glob, try any incorrect folderpath with ${workspaceFolder}/

You will see it does not correctly count the path as invalid